### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by petaalcala**
# Summary

- Added endpoint for sign up(the branch name is wrong 🤣 )
- Added validation middleware
- Added user creation service
- Added migration for remove created_at and updated_at

# Trello card

- https://trello.com/c/qOXKY41U/2-sign-up

### CODE REVIEW

File: migrations/migrations/20191025184730-removeTimestampsFromUser.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
```
Comment (KevinTemes) [inline migrations/migrations/20191025184730-removeTimestampsFromUser.js]: Why did you remove the timestamps?

File: app/validators/user.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    value.match(/([\w_-]*@wolox.[a-z]{3}(.[a-z]*)?)/)
+  ),
+  check('email').custom(email =>
+    User.findOne({ where: { email } }).then(user => {
+      if (user) {
+        throw new Error('E-mail already in use');
+      }
+    })
+  )
+]);
```
Comment (KevinTemes) [inline app/validators/user.js]: The validator seems fine to me, but keep in mind that we usually store this kind of pre-controller validations in a `/middlewares` folder, not in `/validators`

**Review by KevinTemes — CHANGES_REQUESTED** [review]

File: migrations/migrations/20191025184730-removeTimestampsFromUser.js (hunk 1 above)
Comment (petaalcala) [reply migrations/migrations/20191025184730-removeTimestampsFromUser.js]: Cause they were disabled. I will try keep them.

**Review by KevinTemes — APPROVED** [review]
